src/cmd/action.go

```diff
@@ -0,0 +1,171 @@
+package cmd
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/creasty/defaults"
+	"github.com/opslevel/opslevel-go/v2023"
+
+	"github.com/opslevel/cli/common"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var actionType string
+
+var createActionCmd = &cobra.Command{
+	Use:   "action --type=$ACTION_TYPE",
+	Short: "Create an action",
+	Long:  "Create an action",
+	Example: `
+cat << EOF | opslevel create action --type=webhook -f -
+name: "Page The On Call"
+description: "Pages the On Call"
+webhookUrl: "https://api.pagerduty.com/incidents"
+httpMethod: "POST"
+headers:
+  accept: "application/vnd.pagerduty+json;version=2"
+  authorization: "Token token=XXXXXXXXXXXXX"
+  from: "someone@example.com"
+payload: |
+  {
+      "incident": {
+          "type": "incident",
+          "title": "{{manualInputs.IncidentTitle}}",
+          "service": {
+            "id": "{{ service | tag_value: 'pd_id' }}",
+            "type": "service_reference"
+          },
+          "body": {
+            "type": "incident_body",
+            "details": "Incident triggered from OpsLevel by {{user.name}} with the email {{user.email}}. {{manualInputs.IncidentDescription}}"
+          }
+      }
+  }
+EOF`,
+	Run: func(cmd *cobra.Command, args []string) {
+		switch actionType {
+		case "webhook":
+			input, err := readWebhookActionCreateInput()
+			cobra.CheckErr(err)
+			result, err := getClientGQL().CreateWebhookAction(*input)
+			cobra.CheckErr(err)
+			fmt.Printf("created webhook action: %s\n", result.Id)
+		default:
+			err := errors.New("unknown action type: " + args[0])
+			cobra.CheckErr(err)
+		}
+	},
+}
+
+var getActionCmd = &cobra.Command{
+	Use:        "action ID",
```

Comment:
```suggestion
	Use:        "action ID|ALIAS",
```
I think its safe to use ID or Alias on this command

Reply:
Added this change, also for trigger definitions and the delete() operations.